pages/index.html

```diff
@@ -28,7 +28,10 @@ <h2>Welcome to Fedora Silverblue 36!</h2>
     </ul>
   </p>
 
-  <p>As always, before installing or upgrading your system to Silverblue 36, make sure to check the latest known <a href="https://ask.fedoraproject.org/tags/c/common-issues/141/none/f36/l/latest">issues</a>.</p>
+  <p>
+    As always, before installing or upgrading your system to Silverblue 36, make sure to check the latest known <a href="https://ask.fedoraproject.org/tags/c/common-issues/141/none/f36/l/latest">issues</a>.
+    To point to issues and suggestions you have with Fedora Silverblue use the <a href="https://github.com/fedora-silverblue/issue-tracker/issues">Silverblue issue tracker</a>.
```

Comment:
Maybe something like:
```suggestion
    See also the <a href="https://github.com/fedora-silverblue/issue-tracker/issues">Silverblue issue tracker</a> that tracks issues specific to Fedora Silverblue.
```